src/Php72/Php72.php

```diff
@@ -80,4 +83,31 @@ public static function php_os_family()
 
         return isset($map[PHP_OS]) ? $map[PHP_OS] : 'Unknown';
     }
+
+    public static function spl_object_id($object)
+    {
+        if (null === self::$hashMask) {
```

Comment:
actually hashMask is calculated without usage of input param here`$object`. maybe extract it to separated function and then 

```php
public static function spl_object_id($object)
{
    return self::getHashMask() ^ hexdec(substr(spl_object_hash($object), self::$hashOffset, PHP_INT_SIZE));
}
```
?